core/templates/pages/admin-page/jobs-tab/admin-jobs-tab.directive.ts

```diff
@@ -41,79 +40,70 @@ angular.module('oppia').directive('adminJobsTab', [
       controller: [function() {
         var ctrl = this;
         ctrl.showJobOutput = function(jobId) {
-          var adminJobOutputUrl = UrlInterpolationService.interpolateUrl(
-            ADMIN_JOB_OUTPUT_URL_TEMPLATE, {
-              jobId: jobId
+          AdminBackendApiService.showJobOutputAsync(jobId)
+            .then(function(response) {
+              ctrl.showingJobOutput = true;
+              ctrl.jobOutput = response || [];
+              ctrl.jobOutput.sort();
```

Comment:
Ditto*

Reply:
done

Reply:
![Screenshot from 2020-12-28 17-18-24](https://user-images.githubusercontent.com/54100702/103212479-e25dfb80-4930-11eb-8889-c23586bf8ebe.png)
Tried using .sort() in admin-backend-api.service rather than admin-jobs-tab.directive,
its showing this weird error in FE test so reverted the change

Reply:
Firstly make sure that the output is a list and then call sort*

Reply:
> Firstly make sure that the output is a list and then call sort*

@DubeySandeep 
Already tried, I type casted the returning array,still test case failing,
Instead of hitting backemd-api should we mock response or something?
![Screenshot from 2020-12-30 21-31-13](https://user-images.githubusercontent.com/54100702/103365918-432f3480-4ae7-11eb-8c67-1db353220f67.png)
![Screenshot from 2020-12-30 21-35-40](https://user-images.githubusercontent.com/54100702/103365919-45918e80-4ae7-11eb-8ef3-0791435f8d70.png)

Reply:
Or maybe we could remove .sort()
What I've seen is the array is either returning null or it is having one element.

Reply:
`jobOutput.sort() || []` is incorrect. Can you please check why the tests are failing? The test should not fail unexpectedly.

Reply:
(I'll approve this PR once this thread is resolved.)